README.md

```diff
@@ -22,12 +22,13 @@ You need to install both [vim-lsp](https://github.com/prabirshrestha/vim-lsp) an
 
 ### Notice
 
-If you use plugin manager that is merging  plugins (ex. dein), Please setting stop merging work(ex. dein / merged = 0).
+If you use plugin manager that is merging  plugins (ex. dein), Please setting stop merging work(ex. dein / merged = 0) or set `g:lsp_settings_servers_dir` option.
```

Comment:
> set `g:lsp_settings_servers_dir` option to set a different directory from the vim-lsp's default.

Reply:
Thanks.
I updated with force-push and remove duplicated `set`.

```diff
-set `g:lsp_settings_servers_dir` option to set a different directory from the vim-lsp's default.
+set `g:lsp_settings_servers_dir` option to a different directory from the vim-lsp's default.
```